src/server/game/AI/CoreAI/UnitAI.cpp

```diff
@@ -405,8 +408,3 @@ bool FarthestTargetSelector::operator()(Unit const* target) const
 
     return true;
 }
-
-void SortByDistanceTo(Unit* reference, std::list<Unit*>& targets)
```

Comment:
This function was added to remove the need to include Object in UnitAI.h, don't remove it.